demo/a2a-oid4vp/.env.example

```
@@ -1,6 +1,7 @@
 OPENAI_API_KEY=your_api_key_here
 DEMO_AGENT_PORT=10003
+DEMO_AGENT_AUTH_TIMEOUT_MS=120000
 CLI_CLIENT_PORT=3010
 IDENTITY_SERVICE_URL=http://localhost:3000
 IDENTITY_SERVICE_ACCESS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlcyI6WyJBZG1pbiJdLCJuYW1lIjoiZGVtbyIsInR5cGUiOiJhY2Nlc3MiLCJpYXQiOjE3NzE5ODA5MjAsImV4cCI6MzI4NzU5ODA5MjAsImF1ZCI6Ikhla2EgSWRlbnRpdHkgU2VydmljZSIsImlzcyI6Ikhla2EiLCJzdWIiOiI1YjQwZDIwMi0yMmI2LTQ2ZDMtOWY0NC04YmY3ZjEwYzNmNGUifQ.j8v7QFYkpwgtPbDU-aIMayMsMo_vLQ9v9h3dBELqBeo
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IDENTITY_SERVICE_ACCESS_TOKEN key should go before the IDENTITY_SERVICE_URL key

(UnorderedKey)